### DISCUSSION TITLE
解决Canvas绘制模糊问题 - 细节个人博客 - 一个正在默默努力的前端菜鸟的个人博客网站

### DISCUSSION THREAD

**Discussion by giscus** [humandetail/squad-blog-frontend#9] [Announcements]
# 解决Canvas绘制模糊问题 - 细节个人博客 - 一个正在默默努力的前端菜鸟的个人博客网站

细节个人博客是一个记录学习笔记以及前端技术的原创独立的个人博客(Humandetail.com)

https://www.humandetail.com/posts/yAwGIx7zWx

**Comment by libondev** [comment]
果然够水

**Reply by humandetail** [reply]
每年都得水一篇才过瘾
